Source/websocket/WebTransfer.h

```diff
@@ -377,8 +392,23 @@ namespace Web {
             }
         }
 
-        IS_MEMBER_AVAILABLE(Hash, hasHash);
+        IS_MEMBER_AVAILABLE(LoadHash, hasLoadHash);
```

Comment:
Why do we need these notifications? I assume it is debug code ?

Reply:
Since hash Loading for large file is taking a long time, so just wanted to inform the caller a progress or indication about it

Reply:
removed